python/paddle/tensor/math.py

```diff
@@ -855,6 +855,45 @@ def add_n(inputs, name=None):
     return out
 
 
+def trunc(x, name=None):
+    '''
+    Returns a new tensor with the truncated integer values of input.
+    Args:
+        x (Tensor): The input tensor, it's data type should be int, float, double.
+        name (str, optional): Name for the operation (optional, default is None). For more information, please refer to :ref:`api_guide_Name`.
+    
+    Returns:
+        Tensor: The product Tensor.
+    
+    Examples:
+        .. code-block:: python
+            import paddle
+            import numpy as np
```

Comment:
You can use `paddle.rand()` directly instead of `numpy.random.random`

Reply:
Done, thanks!